src/services/UserService.ts

```diff
@@ -111,7 +104,7 @@ const getTenant = () => KC.tokenParsed?.tenant
 const getRoles = () =>
   KC.tokenParsed?.resource_access[keycloakConfig.clientId]?.roles.concat(
     KC.tokenParsed?.resource_access[keycloakConfigSemantic.clientId]?.roles,
-    KC.tokenParsed?.resource_access[keycloakConfigDigitalTwin.clientId]?.roles
+    ''
```

Comment:
@oyo Can you please re-check if I can concatenate roles with empty string or can you suggest me with any better logic?